### PR TITLE
imports fixed - #4

### PR DESCRIPTION
**Pull request by zouhir**
please advise if there's no need for tests for dsm `require` and we'd rather take them off. :)

### CODE REVIEW

File: source/test/test.js [hunk 1]
```diff
@@ -35,6 +35,24 @@ const mockOptions = ({
   component, description, actionStates
 });
 
+test('modules & package specs', nest => {
+  nest.test('dsm function exposed', assert => {
+    const msg = 'current file is AMD, not ready to be commonjs node package';
```
Comment (ericelliott) [inline source/test/test.js]: Change msg to `'should export commonjs module`.

File: source/test/test.js [hunk 2]
```diff
@@ -35,6 +35,24 @@ const mockOptions = ({
...
+    const expected = 'function';
+    const actual = typeof require('../dsm');
+    assert.same(actual, expected, msg);
+    assert.end();
+  });
+
+  nest.test('dsm object exposed', assert => {
+    const msg = 'expose dsm property so `import { dsm }` won\'t fail';
```
Comment (ericelliott) [inline source/test/test.js]: `'should expose [...]'`

**Comment by zouhir** [comment]
messages have been updated

**Comment by ericelliott** [comment]
Cool... will publish tonight.

**Pull request merged**